static/src/javascripts-legacy/bootstraps/standard/main.js

```diff
@@ -85,53 +85,6 @@ define([
             }
         });
 
-        /*
```

Comment:
@SiAdcock does this whole delete seem right to you? i think it is in terms what the switch affected, but seems like to go?